### DISCUSSION TITLE
WingetUI runs but comes up empty.  buttons do nothing but scan for nothing.   tried to get it going for a day and I officially am  totally stuck.

### DISCUSSION THREAD

**Discussion by chowderhead** [Devolutions/UniGetUI#1896] [Q&A]
maybe some more useful info most looks totally normal

C:\Windows\system32>winget --info
Windows Package Manager v1.7.10582
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.17763.5329
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.22.10582.0

Winget Directories
Logs %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User) %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User) %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root C:\Program Files\WinGet\Packages
Portable Package Root (x86) C:\Program Files (x86)\WinGet\Packages
Installer Downloads %USERPROFILE%\Downloads

Links
Privacy Statement https://aka.ms/winget-privacy
License Agreement https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting State
LocalManifestFiles Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride Disabled
LocalArchiveMalwareScanOverride Disabled

my log has a bunch of python errors maybe mean something to you. Reinstalled python - no effect.
[errors.log.txt](https://github.com/marticliment/WingetUI/files/14582775/errors.log.txt)

any help would be appreciated.  Ive tried REALLY HARD with over 30 webpages and 10 youtube videos but the FACT REMAINS.  python is failing and so the app sits dormant.  at least it appears that way.  but as I said Im at my technical limit and pretty exhausted with this.  It all started with needing an app to run my audio system.  I got the app via winget s small success.  but now UI just sits.
RK

WINGET works.  ive installed apps etc, wingetUI does not work and I cant firgure out why.

**Comment by marticliment** [comment]
Please test this beta release:

https://github.com/marticliment/WingetUI/releases/download/3.0.0-beta1/WingetUI.Installer.exe
